src/Nri/Ui/Css/V1.elm

```diff
@@ -11,10 +11,10 @@ import Css exposing (Style, batch)
 
 {-| Apply a style only if a condition is `True`.
 -}
-styleIf : (() -> Style) -> Bool -> Style
-styleIf view condition =
+styleIf : Bool -> Style -> Style
+styleIf condition view =
```

Comment:
how would you feel about this instead:

```elm
styleIf : Bool -> List Style -> Style
styleIf cond stuff = batch (if cond then stuff else [])
```

The result would look like:

```elm
styleIf (Flags.isEnabled model.someFeatureFlag)
    [ backgroundColor red
    , float left
    , position absolute
    , blink True
    ]
```

(with, uh, better styles but you get the idea)

It might be better to flip the arguments, but I don't see us using this a lot in pipelines so I don't think it matters and we can define a local `flip` for it if we need it.

Reply:
Yeah, I like that better.  More than likely there will be a handful of associated styles you'd wish to set.

Reply:
This is not blocking, but I prefer adding less DSL like constructs and use tools that new people are already familiar.
```elm
Css.batch <| if (Flags.isEnabled model.someFeatureFlag) then
    [ backgroundColor red
    , float left
    , position absolute
    , blink True
    ]
else
  []
```
Or if we really are annoyed by this, I could see:
```elm
if (Flags.isEnabled model.someFeatureFlag) then
   Css.batch [ backgroundColor red
    , float left
    , position absolute
    , blink True
    ]
else
  Nri.Css.noStyles
```

Reply:
Adding `styleIf` opens up the door for adding more and more DSL-thingies. See https://package.elm-lang.org/packages/elm-community/html-extra/latest/Html-Extra
Like later people need to learn `styleJust` `styleNothing` `styleUnless` and so on. 
Where as adding `noStyles` can be used in all these cases with "normal" elm syntax.